fedgov-cv-ISDcreator-webapp/src/main/webapp/index.html

```diff
@@ -406,7 +406,7 @@ <h4 class="list-group-item-heading">Allowed Lane Attributes</h4>
 							  <div class="col-sm-7"><input class="form-control" id="intersection" placeholder="0 to 65535" data-parsley-pattern="^[0-9]+$" data-parsley-required="true" data-parsley-range="[0, 65535]" data-parsley-pattern-message="This value must be an integer."></div>
 							  <div class="col-sm-1"><i class="fa fa-question-circle" aria-hidden="true" tag="intersection_id"></i></div>
 							</div>
-							<div class="row major_version extra_rga_field">
+							<!-- <div class="row major_version extra_rga_field">

```

Comment:
Can you remove all the commented out code? Just realized it's better to remove now if we won't be using them later anyway.

Reply:
@jwillmartin Removed all the commented out code